.editorconfig

```diff
@@ -5,17 +5,16 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
-trim_trailing_whitespace = true
 charset = utf-8
-
-[*.{md,css,html}]
```

Comment:
We still have some `html` and `css` files. Why did you remove the rules for them?

Reply:
I admit I'm not sure what happened here. Reverting changes -- thanks!